Gruntfile.js

```diff
@@ -7,7 +7,7 @@
 var path = require('path'),
     getCodeVersion = require('silvermine-serverless-utils/src/get-code-version'),
     join = path.join.bind(path),
-    nodeSass = require('node-sass');
+    nodeSass = require('sass');
```

Comment:
Sorry, just saw this when submitting the review. We probably should also rename `nodeSass` since it's not node-sass anymore.